config/config.go

```diff
@@ -171,6 +172,13 @@ func OptionKVOpts(opts map[string]string) Option {
 	}
 }
 
+// OptionRestoreNetwork function returns an option setter for restore network
+func OptionRestoreNetwork(restore bool) Option {
```

Comment:
How and when is this option set from the daemon ?
Will this be set only in cases where the daemon is restarted (with running containers ?) or also in cases when the host reboots ungracefully ?